app/jobs/activities/dispatch_webhooks_job.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Activities
+  class DispatchWebhooksJob < ApplicationJob
+    queue_as :high_priority
+
+    def perform(activity)
+      webhooks =
+        Webhook.active.where(
+          activity_filter_id:
+            Activities::ActivityFilterMatchingService.new(activity).activity_filters.select(:id)
+        )
+
+      webhooks.each do |webhook|
+        Activities::SendWebhookJob.perform_later(webhook, activity)
```

Comment:
is it really needed to schedule another jobs?

Reply:
Since some of them can fail, we would need to add additional exception handling in the loop here (something like `rescue nil` at the end of the line to ignore exceptions), if performed inline. It seemed cleaner to just dispatch them as separate jobs and keep the dispatch job simpler, and keep exception handling in one place.

Reply:
One reason is also webhook retries, which wouldn't work if they were all performed here.